package.json

```diff
@@ -10,7 +10,8 @@
   "dependencies": {
     "core-js": "^3.6.5",
     "vue": "^3.0.0",
-    "vue-router": "^4.0.0-0"
+    "vue-router": "^4.0.0-0",
```

Comment:
Is there a reason for a router on a single page site?

Reply:
The router was implemented much earlier on (by me) because I thought we'd have a multi-page design. It's been left in because of the blog talks (and it doesn't hurt the website really). Sanaa just added a comma to this line :)